api/app/signals/apps/users/v1/views/permissions.py

```diff
@@ -0,0 +1,19 @@
+from datapunt_api.rest import DatapuntViewSet
+from django.contrib.auth.models import Permission
+from rest_framework.permissions import DjangoModelPermissions
+
+from signals.apps.api.generics.permissions import SIAPermissions
+from signals.apps.users.v1.serializers.permissions import PermissionSerializer
+from signals.auth.backend import JWTAuthBackend
+
+
+class PermissionViewSet(DatapuntViewSet):
+    queryset = Permission.objects.prefetch_related(
+        'content_type',
+    ).all()
+
+    authentication_classes = (JWTAuthBackend,)
+    permission_classes = (SIAPermissions & DjangoModelPermissions,)
```

Comment:
Ok, for now --- but this may be a problem in the future. People who can edit the Permissions model will need the newly defined SIA specific permissions AND the Django model permission (which we intended to hide - but we cannot hide if they are needed).

see UX desing

Reply:
I think this may be OK because of the lazy nature of querysets.

Reply:
This is a starting point because we will implement this in steps, like we discussed. So this will change in the future. So for now not an issue.